cumulusci/tasks/bulkdata/tests/test_bulkdata.py

```diff
@@ -1604,3 +1605,22 @@ def test_split_dependencies__interviews_for_cycles(self, prompt):
                 },
             ),
         )
+
+
+class TestGenerateAndLoadData(unittest.TestCase):
+    def test_batching_no_remainder(self):
+        batches = list(generate_batches(num_records=20, batch_size=10))
+        assert batches == [(10, 0), (10, 1)]
+
+        batches = list(generate_batches(num_records=20, batch_size=5))
+        assert batches == [(5, 0), (5, 1), (5, 2), (5, 3)]
+
+        batches = list(generate_batches(num_records=3, batch_size=1))
+        assert batches == [(1, 0), (1, 1), (1, 2)]
+
+        batches = list(generate_batches(num_records=3, batch_size=3))
+        assert batches == [(3, 0)]
+
+    def test_batching_with_remainder(self):
+        batches = list(generate_batches(num_records=20, batch_size=7))
+        assert batches == [(7, 0), (7, 1), (6, 2)]
```

Comment:
Move this to test_utils so it parallels the structure of the code being tested.

Reply:
I had done that but forgotten to push it out.

https://github.com/SFDO-Tooling/CumulusCI/blob/529f491e088e1e9c2dd6044609e9590c0868bc63/cumulusci/tasks/bulkdata/tests/test_utils.py#L60-L76